### PR TITLE
Add Holesky testnet

### PR DESCRIPTION
**Pull request by jochem-brouwer**
See: https://github.com/eth-clients/holesky

I have asked for confirmation of fork hashes on R&D (they should be right).

Also tested client startup (which works) with the `--network holesky` flag.

### CODE REVIEW

File: packages/common/src/chains.ts [hunk 1]
```diff
@@ -448,4 +448,122 @@ export const chains: ChainsDict = {
...
+        forkHash: '0x01d65a8c',
+      },
+      {
+        name: 'london',
+        block: 0,
+        forkHash: '0x01d65a8c',
+      },
+      {
+        // The forkHash will remain same as mergeForkIdTransition is post merge,
+        // terminal block: https://sepolia.etherscan.io/block/1450408
```
Comment (acolytec3) [inline packages/common/src/chains.ts:528]: Is this comment (or at least this link) still applicable?

**Review by g11tech — APPROVED** [review]
lgtm

**Review by acolytec3 — DISMISSED** [review]
LGTM.  One minor question but not a barrier to merging

File: packages/common/src/chains.ts (hunk 1 above)
Comment (jochem-brouwer) [reply packages/common/src/chains.ts:528]: No this is not applicable in this case, thanks :smile:

**Review by g11tech — APPROVED** [review]

**Pull request merged**